lms/templates/problem.html

```diff
@@ -15,13 +15,13 @@ <h2 class="problem-header">
     <input type="hidden" name="problem_id" value="${ problem['name'] }" />
 
     % if check_button:
-    <button class="check ${ check_button }" data-checking="${ check_button_checking }" data-value="${ check_button }"><span class="check-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span></button>
+    <button class="check ${ check_button }" data-checking="${ check_button_checking }" data-value="${ check_button }"><span class="check-label button-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span></button>
```

Comment:
@ahsan-ul-haq is there a particular reason that you included an additional span here?  Could you just add the new class on the ```<button>``` element for all of these?